WordPress/WordPressShareExtension/ShareViewController.swift

```diff
@@ -139,6 +147,17 @@ class ShareViewController: SLComposeServiceViewController {
         pushConfigurationViewController(pickerViewController)
     }
     
+    private func loadTextViewContent() {
+        extensionContext?.loadWebsiteUrl { url in
+            dispatch_async(dispatch_get_main_queue()) {
+                let current = self.contentText ?? String()
+                let source  = url?.absoluteString ?? String()
+                
+                self.textView.text = "\(current)\n\n\(source)"
```

Comment:
If no URL or only a URL exists lets make sure no blank space is put into the text box.
![img_9856](https://cloud.githubusercontent.com/assets/373903/14891601/fa4522f4-0d2c-11e6-9e3f-4113e9395bce.jpg)